assembly/benches/sha256.rs

```diff
@@ -0,0 +1,30 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::time::Duration;
+use miden_assembly::{self, Assembler};
+
+
+fn script_compilation(c: &mut Criterion) {
+    let mut group = c.benchmark_group("sha256");
```

Comment:
Maybe change the name here to "script_compilation"?

Reply:
Done!